### PR TITLE
Build watcher

### PR DESCRIPTION
**Pull request by jamesgpearce**
use `./build -w` to have it watch the moves, the transpilation, and the browserification on any relevant file change

### CODE REVIEW

File: standalone/build.js [hunk 1]
```diff
@@ -95,18 +105,45 @@ function build() {
...
+      cache: {},
+      packageCache: {},
+      verbose: true,
     }
   );
 
-  browserifyJob.on('error', exitOnError);
-  const bundle = browserifyJob.bundle();
-  bundle.on('error', exitOnError);
+  bundler
```
Comment (bolinfest) [inline standalone/build.js]: I'm pretty sure this ignore call is gone now?

Comment (jamesgpearce) [reply standalone/build.js]: (y)

**Comment by jamesgpearce** [comment]
doesn't look like I can merge, so gfi

**Comment by jamesgpearce** [comment]
adding new keymaps for chrome 54

**Comment by jamesgpearce** [comment]
worth merging...

**Pull request merged**